core/internal/cltest/factories.go

```diff
@@ -442,9 +435,41 @@ func EmptyCLIContext() *cli.Context {
 	return cli.NewContext(nil, set, nil)
 }
 
-func CreateJobRunWithStatus(t testing.TB, store *store.Store, j models.JobSpec, status models.RunStatus) models.JobRun {
-	initr := j.Initiators[0]
-	run := j.NewRun(initr)
+// NewJobRun returns a newly initialized job run for test purposes only
+func NewJobRun(job models.JobSpec) models.JobRun {
```

Comment:
Minor point, since the function is in cltest, is the note about its purpose helpful?